kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java

```diff
@@ -438,6 +438,18 @@ public class BigQuerySinkConfig extends AbstractConfig {
       "The time partitioning type to use when creating tables. "
           + "Existing tables will not be altered to use this partitioning type.";
 
+  public static final String MAX_RETRIES_CONFIG = "max.retries";
+  private static final ConfigDef.Type MAX_RETRIES_TYPE = ConfigDef.Type.INT;
+  private static final int MAX_RETRIES_DEFAULT = 10;
+  private static final ConfigDef.Validator MAX_RETRIES_VALIDATOR = ConfigDef.Range.atLeast(1);
+  private static final ConfigDef.Importance MAX_RETRIES_IMPORTANCE = ConfigDef.Importance.MEDIUM;
+  private static final String MAX_RETRIES_DOC = "The maximum number of times to retry on errors before failing the task.";
+
+  public static final String ENABLE_RETRIES_CONFIG = "enableRetries";
+  private static final ConfigDef.Type ENABLE_RETRIES_TYPE = ConfigDef.Type.BOOLEAN;
+  public static final Boolean ENABLE_RETRIES_DEFAULT = false;
```

Comment:
Let's enable it by default on both cloud and cp.

Reply:
The idea is to make sure this doesn't cause regression for other customers. Maybe worthwhile to verify this for current customer first and then enable it after we have some confidence. Thoughts?

Reply:
Updated the release plan in PR description.